lib/index.js

```diff
@@ -15,11 +15,21 @@ const conditionalTranslate = (key, t) => {
   return result;
 };
 
-const getFields = (key, t) => ({
+const changeValidate = (optional) => {
+  let validateItems = ['required', 'postcode'];
+
+  if (optional) {
+    validateItems = ['postcode'];
+  }
+  return validateItems;
+};
```

Comment:
This is very long-winded, and largely redundant. See comment on `getFields` below.